src/util.js

```diff
@@ -16,6 +16,31 @@ function loadQuery(query) {
   return fs.readFileSync(path.resolve(__dirname, 'queries', `${query.replace(/^\//, '')}.sql`)).toString();
```

Comment:
Why sync file read?  

What about error handling?

Reply:
the sync is necessary; because everything afterwards relies on it; so might as well block.

Reply:
is there a better way to do this?

Reply:
Use async/await. 

`sync` vs `async` is not about whether some of your tasks meed to be executed in sequence or can be run in parallel. Execution in a node vm is essentially single-threaded. When you are calling a synchronous method you are basically blocking the entire process until the method returns. When you are calling  an asynchronous method node's `libuv` based event loop still processes IO and other events. 

It's good practice to avoid sync (i.e. blocking) calls whenever possible.

Reply:
Done.

Reply:
This is done! Thanks for the great information; @stefan-guggisberg